apps/src/templates/teacherDashboard/ScriptSelector.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
I just noticed this is a js file not jsx, on purpose?

Reply:
either is fine. I think I usually use .js